routers/web/repo/issue.go

```diff
@@ -992,7 +992,9 @@ func NewIssuePost(ctx *context.Context) {
 	}
 
 	log.Trace("Issue created: %d/%d", repo.ID, issue.ID)
-	ctx.Redirect(ctx.Repo.RepoLink + "/issues/" + fmt.Sprint(issue.Index))
+	ctx.JSON(http.StatusOK, map[string]interface{}{
```

Comment:
How do things like

```
	if util.IsEmptyString(form.Title) {
		ctx.RenderWithErr(ctx.Tr("repo.issues.new.title_empty"), tplIssueNew, form)
		return
	}
```

on line 963 behave now that this form is submitted with AJAX?